### PR TITLE
7733 new design for features page

### PR DESCRIPTION
**Pull request by denpira**
https://codio.myjetbrains.com/youtrack/issue/codio-7733

### CODE REVIEW

File: source/features/index.html.erb [hunk 1]
```diff
@@ -14,114 +14,191 @@ fixed: true
...
-          </h3>
-          <div class="section-block-copy">
-            <p>
-              We’ve co-developed over 50 courses with Oxford, Cambridge and RSA
-              (OCR) — all mapped to the UK Computer Science national curriculum.
-            </p>
-            <%= link_to 'Learn more', '/features/curriculum', class: 'button button-large learn-more' %>
-          </div>
-        </div>
+    </div><!--
```
Comment (AndreyNikitin) [inline source/features/index.html.erb]: useless comment

File: source/features/index.html.erb [hunk 2]
```diff
@@ -14,114 +14,191 @@ fixed: true
...
+      <div class="action-button animated">
+        <%= link_to 'Learn More', '/docs/content/authoring/assessments/', class: 'button button-large blue-button' %>
+      </div>
+      <h4 class="quote-title">David Croft, Lecturer, Coventry University says:</h4>
+      <div class="speech-purple"></div>
+      <div class="quote-text">"Having the instant formative feedback is really useful. And that’s where Codio comes in handy.
+        They click check it in Codio and it goes and tests that their code meets the specifications. It sorts correctly, it
+        searches correctly. It has the error handling and features that we are expecting. It doesn’t have memory leaks."
       </div>
+    </div><!--
```
Comment (AndreyNikitin) [inline source/features/index.html.erb]: useless comment

File: source/features/index.html.erb [hunk 3]
```diff
@@ -14,114 +14,191 @@ fixed: true
...
+    <div class="text-part">
+      <h3 class="title">Curriculum content</h3>
+      <p class="info">We’ve co-developed over 80+ courses with Oxford, Cambridge and RSA (OCR) - all mapped
+        to UK national standards. We are also writing more curriculum to be in line with emerging Computer Science
+        Teachers Association K-12 framework.
+      </p>
+      <div class="action-button animated">
+        <%= link_to 'Learn More', '/features/curriculum/', class: 'button button-large blue-button' %>
       </div>
+    </div><!--
```
Comment (AndreyNikitin) [inline source/features/index.html.erb]: useless comment

**Comment by denpira** [comment]
All comments is necessary.

File: source/features/index.html.erb [hunk 4]
```diff
@@ -14,114 +14,186 @@ fixed: true
...
+      <div class="action-button animated">
+        <%= link_to 'Learn More', '/docs/content/authoring/assessments/', class: 'button button-large blue-button' %>
+      </div>
+      <h4 class="quote-title">David Croft, Lecturer, Coventry University says:</h4>
+      <div class="speech-purple"></div>
+      <div class="quote-text">"Having the instant formative feedback is really useful. And that’s where Codio comes in handy.
+        They click check it in Codio and it goes and tests that their code meets the specifications. It sorts correctly, it
+        searches correctly. It has the error handling and features that we are expecting. It doesn’t have memory leaks."
       </div>
+    </div><div class="image-part">
```
Comment (mloginov) [inline source/features/index.html.erb]: new tag from new line
```
</div>
<div class="image-part">
```

File: source/features/index.html.erb [hunk 5]
```diff
@@ -14,114 +14,186 @@ fixed: true
...
+    <div class="text-part">
+      <h3 class="title">Curriculum content</h3>
+      <p class="info">We’ve co-developed over 80+ courses with Oxford, Cambridge and RSA (OCR) - all mapped
+        to UK national standards. We are also writing more curriculum to be in line with emerging Computer Science
+        Teachers Association K-12 framework.
+      </p>
+      <div class="action-button animated">
+        <%= link_to 'Learn More', '/features/curriculum/', class: 'button button-large blue-button' %>
       </div>
+    </div><div class="image-part">
```
Comment (mloginov) [inline source/features/index.html.erb]: new tag from new line

**Comment by denpira** [comment]
Done

File: source/stylesheets/pages/features/_index.scss [hunk 6]
```diff
@@ -35,122 +88,437 @@ body.features_index .header-block {
...
+    .image-part {
+      float: left;
+      vertical-align: top;
+      width: 100%;
+      
+      img {
+        margin-top: 1.414rem;
+        width: 100%;
+        height: 100%;
+        //max-height: 348px;
```
Comment (AndreyNikitin) [inline source/stylesheets/pages/features/_index.scss]: remove commented code

File: source/stylesheets/pages/features/_index.scss [hunk 7]
```diff
@@ -1,30 +1,83 @@
...
+body.features {
+  header .container,
+  footer .container {
+    @include breakpoint($max-width) {
+      margin-left: auto;
+      margin-right: auto;
+    }
+  }
+  .content {
+    margin: 0 auto 0 auto;
```
Comment (AndreyNikitin) [inline source/stylesheets/pages/features/_index.scss]: margin: 0 auto;

File: source/stylesheets/pages/features/_index.scss [hunk 8]
```diff
@@ -1,30 +1,83 @@
...
+    max-width: $max-width;
+  }
+}
+
+body.features_index .content {
+  margin: 0 auto 0 auto;
```
Comment (AndreyNikitin) [inline source/stylesheets/pages/features/_index.scss]: margin: 0 auto;

**Comment by AndreyNikitin** [comment]
:+1:

File: source/stylesheets/pages/features/_index.scss (hunk 6 above)
Comment (denpira) [inline source/stylesheets/pages/features/_index.scss]: Done

File: source/stylesheets/pages/features/_index.scss (hunk 7 above)
Comment (denpira) [inline source/stylesheets/pages/features/_index.scss]: Done

File: source/stylesheets/pages/features/_index.scss [hunk 9]
```diff
@@ -1,30 +1,83 @@
...
-    font-size: 48px;
-    font-weight: 300;
+    font-size: 42px;
+    font-weight: 500;
     text-align: center;
     animation-delay: .4s;
-
     @include breakpoint($break-tablet) {
-      margin-top: 2rem;
+      margin: 2rem auto 2rem auto;
```
Comment (denpira) [inline source/stylesheets/pages/features/_index.scss]: Done

**Pull request merged**